Modules/_datetimemodule.c

```diff
@@ -1873,6 +1878,11 @@ wrap_strftime(PyObject *object, PyObject *format, PyObject *timetuple,
     pnew = PyBytes_AsString(newfmt);
     usednew = 0;
 
+    PyObject *strftime = _PyImport_GetModuleAttrString("time", "strftime");
```

Comment:
`strftime` holds now a reference, which should be released before leaving the function. Move `Py_DECREF(strftime)` to the cleanup code after the `Done` label.